build/dependencies.props

```diff
@@ -97,6 +97,7 @@
     <HealthCheckUI>2.2.32</HealthCheckUI>
     <HealthCheckUIClient>2.2.4</HealthCheckUIClient>
     <HealthCheckPublisherAppplicationInsights>2.2.4</HealthCheckPublisherAppplicationInsights>
+    <HealthCheckPublisherDatadog>2.2.0</HealthCheckPublisherDatadog>
```

Comment:
I wasn't sure what the appropriate versioning is for this repo, but it looked like everything was at least 2.2.